assets/scss/main.scss

```diff
@@ -170,6 +157,29 @@ a {
   }
 }
 
+.link {
+  @include link;
+}
+
+.link-no-decoration {
+  color: inherit;
+  text-decoration: none;
+
+  &:hover, &:active, &:visited {
+    text-decoration: none;
+  }
+}
```

Comment:
Just to note: when you just move logic, it is harder to notice any changes in code, so please do not do that unless necessary

Reply:
It was me how told him to do that. Sorry @k911